Makefile

```diff
@@ -64,20 +64,20 @@ start-idp:  ## 🏃 Start the idp for testing jwt
 # Start hosting the application using `sandbox.sh` and enable custom JWT authentication
 start-host: stop-host  ## 🏃 Start the CCF network using Sandbox.sh
 	@echo -e "\e[34m$@\e[0m" || true
-	MEMBER_COUNT=${MEMBER_COUNT} source ./scripts/ccf/sandbox_local/up.sh --build && \
-	source ./scripts/kms/js_app_set.sh && \
-	source ./scripts/kms/constitution_set.sh \
+	MEMBER_COUNT=${MEMBER_COUNT} . ./scripts/ccf/sandbox_local/up.sh --build && \
+	./scripts/kms/js_app_set.sh && \
```

Comment:
Did you mean to spawn a new shell? Is there any env variable being passed out? I guess this is deliberate but just in case...

Reply:
There were no env variables being passes out by this so just calling the script directly is sufficient